content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'endswith()'
+Description: 'Checks whether or not a string ends with the given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `endswith()` string method checks to see if a particular value is attached to the end of the string it is called on and returns a boolean value.
```

Comment:
```suggestion
The `.endswith()` method returns `True` if a particular value is attached to the end of a given string. Otherwise, it returns `False`.
```